images/runtime/Dockerfile

```diff
@@ -42,9 +42,8 @@ WORKDIR /go/src/github.com/cilium/cilium/images/runtime
 RUN --mount=type=bind,readwrite,target=/go/src/github.com/cilium/cilium/images/runtime \
     ./install-runtime-deps.sh
 
-COPY iptables-wrapper /usr/sbin/iptables-wrapper
 RUN --mount=type=bind,readwrite,target=/go/src/github.com/cilium/cilium/images/runtime \
-    ./configure-iptables-wrapper.sh
+    ./iptables-wrapper-installer.sh --no-sanity-check
```

Comment:
Do we need to skip the sanity check? Cilium ships iptables 1.8.4 currently so the script should pass.

```
$ docker run quay.io/cilium/cilium:v1.9.16 iptables -V
iptables v1.8.4 (legacy)
```

Reply:
^ But I guess that means that it doesn't matter at all, we already guarantee the right version so this invocation is good as-is.